logshifter/input.go

```diff
@@ -64,24 +67,39 @@ func (input *Input) read() {
 
 		copy(cp, line)
 
-		if input.statsChannel != nil {
-			input.statsChannel <- Stat{name: "input.read", value: 1.0}
-		}
+		input.record("input.read", 1.0)
 
 		select {
 		case input.queue <- cp:
 			// queued
+			input.record("input.queue", 1.0)
```

Comment:
Could add the new "input.queue" key to the enumeration of stats in the comment for `struct Input`.  Likewise for "input.evict".